mx.sulong/mx_sulong.py

```diff
@@ -573,10 +573,9 @@ def runLLVMTestCases(args=None):
 
 def runTruffleTestCases(args=None):
     """runs the Sulong test suite"""
-    ensureLLVMBinariesExist()
-    ensureDragonEggExists()
+    testsuites.compileSuite(['sulong'])
     vmArgs, _ = truffle_extract_VM_args(args)
-    return unittest(getCommonUnitTestOptions() + ['-Dsulong.ExecutionCount=1000'] + vmArgs + ["com.oracle.truffle.llvm.test.SulongTestSuite"])
+    return unittest(vmArgs + ["com.oracle.truffle.llvm.test.alpha.SulongSuite"])
```

Comment:
I intentionally do not pass any options here.
I feel like we should only pass the options that we actually need to run the suite.
/cc @mrigger